### DISCUSSION TITLE
清晰度问题

### DISCUSSION THREAD

**Discussion by HeartDeer** [ywmoyue/biliuwp-lite#142] [Q&A]
每次打开都需要登录才会有1080p的清晰度，不然就只有360p和480p

**Comment by ywmoyue** [comment]
已有Issue：https://github.com/ywmoyue/biliuwp-lite/issues/31

**Comment by ywmoyue** [comment]
请问下更新4.6.3后是否还会出现已登陆时分辨率只有480p的情况 @HeartDeer

**Reply by HeartDeer** [reply]
谢谢你的解决，目前不再出现这个问题了
